File: app/src/main/java/com/example/watering/investrecord/DBHelper/GroupDBHelper.java
package com.example.watering.investrecord.DBHelper;

import android.content.Context;

/**
 * Created by watering on 17. 10. 21.
 */

@SuppressWarnings("DefaultFileTemplate")
public class GroupDBHelper extends DBHelper {

    public GroupDBHelper(Context context) {
        super(context);
        TABLE_NAME = "tbl_Group";
        COLUMNS = new String [] {"name TEXT UNIQUE"};
    }
}

File: app/src/main/java/com/example/watering/investrecord/dbhelper/GroupDBHelper.kt
package com.example.watering.investrecord.dbhelper

import android.content.Context

/**
 * Created by watering on 17. 10. 21.
 */

class GroupDBHelper(context: Context) : DBHelper(context) {

    init {
        TABLE_NAME = "tbl_Group"
        COLUMNS = arrayOf("name TEXT UNIQUE")
    }
}


